src/types/QueryFilterTypes.ts

```diff
@@ -129,3 +129,19 @@ export interface ParsedFilterInterval {
   end?: string;
   interval?: cql.Interval;
 }
+
+/**
+ * Represents object containing query parameters and their corresponding values.
+ */
+export interface codeFilterPath {
```

Comment:
This seems to be representing a generic hash/dictionary type thing. Not a typescript expert, but... Do we think it would make sense to use something standard, like it looks we could use something like the [Record](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type) utility type, or if that's not a good fit, maybe we could define cross project hash interface utility type?

Reply:
I'm so proud of this community

Reply:
Lauren understood the assignment of my "I encourage you to look into some of the other types you can use in TypeScript" during my talk 😭